src/server_manager/web_app/digitalocean_app/create_server_app.ts

```diff
@@ -0,0 +1,77 @@
+/*
+  Copyright 2018 The Outline Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+import {customElement, html, LitElement, property} from 'lit-element';
+
+import * as server from '../../model/server';
+import {App, NotificationManager} from '../app';
+
+import {Location, OutlineRegionPicker} from '../ui_components/outline-region-picker-step';
+
+// DigitalOcean mapping of regions to flags
+const FLAG_IMAGE_DIR = 'images/flags';
+const DIGITALOCEAN_FLAG_MAPPING: {[cityId: string]: string} = {
+  ams: `${FLAG_IMAGE_DIR}/netherlands.png`,
+  sgp: `${FLAG_IMAGE_DIR}/singapore.png`,
+  blr: `${FLAG_IMAGE_DIR}/india.png`,
+  fra: `${FLAG_IMAGE_DIR}/germany.png`,
+  lon: `${FLAG_IMAGE_DIR}/uk.png`,
+  sfo: `${FLAG_IMAGE_DIR}/us.png`,
+  tor: `${FLAG_IMAGE_DIR}/canada.png`,
+  nyc: `${FLAG_IMAGE_DIR}/us.png`,
+};
+
+@customElement('digital-ocean-create-server')
+export class DigitalOceanCreateServer extends LitElement {
+  @property({type: Function}) localize: Function;
+  @property({type: Object}) app: App = null;
+  @property({type: Object}) digitalOceanRepository: server.ManagedServerRepository = null;
+  @property({type: Object}) notificationManager: NotificationManager = null;
+
+  render() {
+    return html`
+        <outline-region-picker-step id="regionPicker" .localize=${
```

Comment:
I'm wondering if we still need the region picker component. Maybe we could have all the logic here to reduce the layering and simplify the logic.

For example, you can probably get rid of the createLocationModel below.

Reply:
Let me take a stab at the GCP create server app and see if we can reuse the `OutlineRegionPicker`. If we can, then it might be good to keep this it cloud agnostic.

Also, I'm leaning towards the adding the create server progress screen to this component. Can you explain again why you think it should be part of the server view?